app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/EateryCard.kt

```diff
@@ -58,142 +59,17 @@ import com.cornellappdev.android.eatery.ui.theme.GrayOne
-//                        backgroundColor = Color.White
-//                    ) {
-//                        Row(
-//                            modifier = Modifier.padding(
-//                                horizontal = 10.dp,
-//                                vertical = 8.dp
-//                            )
-//                        ) {
-//                            Icon(
-//                                Icons.Outlined.Warning,
-//                                contentDescription = "Closing in 10 min",
-//                                modifier = Modifier.size(ButtonDefaults.IconSize)
-//                            )
-//                            Spacer(Modifier.size(ButtonDefaults.IconSpacing))
-//                            Text(
-//                                text = "Closing in ${xMinutesUntilClosing!!.value} min",
-//                                style = EateryBlueTypography.button
-//                            )
-//                        }
-//                    }
-//                }
-//            }
-//            Column(
-//                modifier = Modifier.padding(10.dp)
-//            ) {
-//                Row(
-//                    horizontalArrangement = Arrangement.SpaceBetween,
-//                    modifier = Modifier.fillMaxWidth()
-//                ) {
-//                    Text(
-//                        text = eatery.name ?: "",
-//                        maxLines = 1,
-//                        overflow = TextOverflow.Ellipsis,
-//                        style = EateryBlueTypography.h5,
-//                        modifier = Modifier
-//                            .weight(1f)
-//                            .padding(end = 30.dp)
-//                    )
-//                    Icon(
-//                        imageVector = if (isFavorite) Icons.Filled.Star else Icons.Outlined.StarOutline,
-//                        tint = if (isFavorite) Yellow else GrayFive,
-//                        modifier = Modifier
-//                            .padding(top = 3.dp)
-//                            .clickable(
-//                                interactionSource = interactionSource,
-//                                indication = rememberRipple(radius = 9.dp),
-//                                onClick = {
-//                                    onFavoriteClick(!isFavorite)
-//                                }
-//                            ),
-//                        contentDescription = null
-//                    )
-//                }
-//
-//                EateryCardPrimaryHeader(eatery = eatery, isCompact = isCompact)
-//                EateryCardSecondaryHeader(eatery = eatery, isCompact = isCompact)
-//            }
-//        }
-//    }
-//}
+enum class EateryCardStyle {
+    DEFAULT, COMPACT, GRID_VIEW
+}
 
 @OptIn(
-    ExperimentalMaterialApi::class, ExperimentalLifecycleComposeApi::class,
-    ExperimentalPermissionsApi::class
+    ExperimentalMaterialApi::class,
 )
 @Composable
 fun EateryCard(
```

Comment:
Where'd walk times go? I'm looking at the app now and don't see walk  times on the card. 

Also, the spacing is definitely a bit strange. There's definitely too much spacing between the title and the "Open until..." text below it. It's subtle, but definitely off.

![image](https://github.com/user-attachments/assets/ee5b211f-48f4-478a-aa9b-59291032b713)

vs.

![image](https://github.com/user-attachments/assets/4aa15dd7-f346-4b10-be1b-21b27d13da4d)

Reply:
Also, my eatery card doesn't seem to be showing the location. I saw in your screenshot that it displays "Carl Becker House", but I don't see that on my end.

Reply:
oops sorry i think I accidentally commented the location out